tests/integration_tests/access_tests.py

```diff
@@ -103,11 +103,7 @@ def tearDownClass(cls):
             db.session.delete(security_manager.find_role(SCHEMA_ACCESS_ROLE))
             db.session.commit()
 
-    def setUp(self):
```

Comment:
My hope was to use a new fixture `auth_as_admin` (which would leverage the `test_client` fixture), however mixing some facets of `pytest` and `unittest` don't work. As a follow up we should remove all `unittest` logic and purely use `pytest` constructs.